app.js

```diff
@@ -0,0 +1,319 @@
+    var slot2_p = document.getElementById('p_flavor');
+
+    // Initialize slot 3 elements
+    //
+    var newUl3  = document.getElementById('newHoppyUl');
+    var newLi3  = document.getElementById('hoppyLi');
+    var slot3_p = document.getElementById('p_hoppy');
+
+    var time0 = 500;
+    var time1 = 1000;
+    var time2 = 1500;
+    var time3 = 2000;
+    var time4 = 2500;
+    var time5 = 4000;
+
+    // Slot 1 fade in with beer placeholder
+    //
+    setTimeout( function() {
+
+      newLi.classList.add('show');
+      newLi.innerHTML = '<img class="placeholder_styling" src= "assets/placeholder_beer.png">';
+
+      // Slot 3 fade out
+      //
+      setTimeout(function() {
+
+        slot3_p.setAttribute('class', 'swing');
+        newLi3.setAttribute('class', 'swing');
+
+        // ok, we are done, lets go to event 3
+        // Slot 3 fade in with beer placeholder
+        //
+        setTimeout( function() {
+
+          newLi3.classList.add('show');
+          newLi3.innerHTML = '<img class="placeholder_styling" src= "assets/placeholder_beer.png">';
+
+          // Slot 2 fade out
+          //
+          setTimeout(function() {
+
+            slot2_p.setAttribute('class', 'swing');
+            newLi2.setAttribute('class', 'swing');
+
+            // Slot 2 fade in with beer RESULT
+            //
+            setTimeout( function() {
+
+              slot2_p.classList.add('show_result');
+
+              // Check if there's a match... if not, display struck out msg
+              //
+              if(!chosenBeer) {
+                var imgPath = 'assets/no_luck.png';
+                var imgId   = 'dummy';
+                slot2_p.innerHTML = "You struck out!";
+                slot2_p.classList.add('blink');
+
+                // If match, display beer photo and name
+                //
+              } else {
+                var imgPath = 'assets/' + chosenBeer.img;
+                var imgId   = 'beer_result_img';
+                slot2_p.classList.add('blink');
+                slot2_p.innerHTML = chosenBeer.name;
+              }
+
+              newLi2.classList.remove('swing');
+              newLi2.classList.add('results_styling');
+              newLi2.classList.add('show_result');
+              newLi2.innerHTML = '<img id=' + imgId + ' class="results_styling" src=' + imgPath + '>';
+
+              // make the beer result glow with background so img doesn't look 'flattened'
+              //
+              $("#beer_result_img").glow({ radius: "16", color:"gold"});
+
+              // we are finally done, leave now
+              //
+              setTimeout( function() {
+                //callback
+                complete();
+
+              }, time5-time4);
+            }, time4-time3);
+          }, time3-time2);
+        }, time2-time1);
+      }, time1-time0);
+    }, time0);
+  },
+}
+
+var type = document.getElementById('type');
+
+type.addEventListener('click', function(e) {
```

Comment:
These event listeners/handlers are a bit repetitive, and could reduced to just one.